networking/src/index.ts

```diff
@@ -1,7 +1,7 @@
 // created from 'create-ts-index'
 
-export * from './AWSMobileApi'
 export * from './ApiClient'
+export * from './AWSMobileApi'
```

Comment:
Can you delete this file locally? If you do and then it won't show up anymore

Reply:
shoot, yup that's been done now. PR updated